modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/subscription/AndesSubscription.java

```diff
@@ -386,7 +386,7 @@ public String getValue(AndesSubscription sub, QueryOptions queryOptions) {
             ("routingKey") {
         @Override
         public String getValue(AndesSubscription sub, QueryOptions queryOptions) {
-            return sub.getStorageQueue().getMessageRouterBindingKey();
+            return sub.getStorageQueue().getMessageRouterBindingKey().toLowerCase();
```

Comment:
How would UI behave otherwise if we need the case sensitivity throughout the product?